### DISCUSSION TITLE
Is HomeKit compatibility required?

### DISCUSSION THREAD

**Discussion by k1200s** [krahabb/meross_lan#81] [General]
Will this method work with Non-HomeKit version devices?

thanks.

**Comment by krahabb** [comment]
Hello @k1200s ,
yes, it is totally agnostic regarding the home-kit features and I know some users are using their home-kit enabled devices with meross_lan without any different behaviour .
Of course every device/version might introduce some random news but overall it should work

**Comment by k1200s** [comment]
Fantastic.  Glad I don’t need to replace all of my devices.  Will get cracking on this today.  Thanks for the reply
